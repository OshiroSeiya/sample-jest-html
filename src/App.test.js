import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('App Snapshot', () => {
  const component = renderer.create(
    <App />
  );
  expect(component).toMatchSnapshot('App is displayed?');
});
