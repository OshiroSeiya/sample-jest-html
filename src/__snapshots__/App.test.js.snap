// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App is displayed? 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    className="App-intro"
    style={
      Object {
        "color": "red",
      }
    }
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
------------HTML PREVIEW---------------
<div class="App">
  <div class="App-header">
    <img
      alt="logo"
      class="App-logo"
      src="logo.svg"
    >
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    class="App-intro"
    style="color:red;"
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
